<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1201</width>
     <height>801</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background-color:qlineargradient(spread:pad, x1:0.275626, y1:0.715909, x2:1, y2:0, stop:0.0689655 rgba(143, 149, 225, 255), stop:1 rgba(255, 255, 255, 255));}</string>
   </property>
   <widget class="QPushButton" name="myRecipes">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>140</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(255, 160, 74);
font:rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>My Recipes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="myMenus">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>230</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(255, 160, 74);
font:rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>My Menus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="chefs">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>320</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(255, 160, 74);
font:rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Chefs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="myProfile">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>410</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(255, 160, 74);
font:rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>My Profile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Log out</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
